src/main/java/gregtechfoodoption/recipe/GTFORecipeMaps.java

```diff
@@ -45,16 +45,28 @@
             .setSlotOverlay(false, false, true, GuiTextures.FURNACE_OVERLAY_1)
             .setProgressBar(GuiTextures.PROGRESS_BAR_ARROW_MULTIPLE, ProgressWidget.MoveType.HORIZONTAL);
 
+    @ZenProperty
+    public static final RecipeMap<ElectricBakingOvenRecipeBuilder> STEAM_BAKING_OVEN_RECIPE = new ElectricBakingOvenRecipeMap<>("steam_baking_oven", new ElectricBakingOvenRecipeBuilder())
```

Comment:
Personally, I don't think this recipe map is actually needed, given that it only is the regular baking oven map with durations halved and steam input, which can be mirrored by other means. The temperature is also borrowed from the ElectricBakingOvenRecipeBuilder, which adds unneeded information.

Reply:
Its difference with normal Baking oven recipe map is that normal baking takes 2 input and fuel while steam oven acts like electric version without temperature value, so it's something in between